tests/demos/checkboxes.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Checkboxes from "@demos/components/checkboxes";
+
+describe("demos/components/checkboxes", () => {
+  it("renders itself correctly", () => {
+    const component = renderer.create(<Checkboxes />)
+    console.log(component)
```

Comment:
so this is not a proper test.

I guess it does not work because `inputRef` is not defined. I think you have to "mock" it like here: https://github.com/Zoapp/react-materialcomponents/blob/c29e019176b9ab5236db07b39022677cc164f836/tests/dialog/dialog.test.js#L29.

I'd say something like this:

```
const createNodeMock = () => ({
    indeterminate: null,
});

const component = renderer.create(
    <Checkboxes />,
    { createNodeMock }
);
```